front-end/src/app/login/login/login.component.ts

```diff
@@ -15,16 +11,9 @@ export class LoginComponent implements OnInit {
   public loginDotGovAuthUrl: string | undefined;
   public localLoginAvailable = false;
 
-  constructor(
-    private loginService: LoginService,
-    private store: Store,
-  ) {}
+  constructor(private loginService: LoginService) {}
 
   ngOnInit() {
-    localStorage.clear();
-    this.loginService.clearUserLoggedInCookies();
-    this.store.dispatch(setCommitteeAccountDetailsAction({ payload: new CommitteeAccount() }));
```

Comment:
It's my understanding that the reason these are here is because currently we can't guarantee that the logout code is called before we get here.  So we want to clear out potentially stale data